iModelCore/ecobjects/PublicApi/ECObjects/ECSchema.h

```diff
@@ -4136,6 +4150,9 @@ friend struct SchemaJsonWriter;
     //!Loops through a schema's classes and properties and removes control characters from their descriptions, display labels, and role labels.
     //! @param[in]  schema  pointer to the schema which will be looped through
     ECOBJECTS_EXPORT static void RemoveInvalidDisplayCharacters(ECSchemaR schema);
+
+    //! The schemaXml will be read to obtain the SchemaKey, ECXML Major and Minor versions.
+    ECOBJECTS_EXPORT static SchemaReadStatus ReadSchemaStub(Utf8StringR schemaXml, SchemaKey& schemaKey, uint32_t& ecXmlMajorVersion, uint32_t& ecXmlMinorVersion);

```

Comment:
On second thought, SchemaStub is a concept we better keep inside this project. What information do you need exactly? If it's only the SchemaKey, then better call it "ReadSchemaKey"

Reply:
For the current problem, I only need to obtain the SchemaKey from the XML.

Reply:
Updated method for extracting only the SchemaKey from the schema XML.